test/services/credential-service/credentials.test.js

```diff
@@ -6,97 +6,120 @@ const credentialService = services.credential;
 const userService = services.user;
 const db = require('../../../lib/db');
 
-describe('Credential service tests', () => {
-  describe('Credential tests', () => {
-    let credential;
-    const username = 'someUser';
+describe('Credential tests', () => {
+  const username = 'someUser';
+  const credential = {
+    secret: 'password'
+  };
 
-    before(() => db.flushdb());
+  function insertCredential(policy) {
```

Comment:
Well technically it's not a policy, it's a credential type
```suggestion
  function insertCredential(type) {
```